wrapanapi/systems/virtualcenter.py

```diff
@@ -243,10 +284,132 @@ def _get_resource_pool(self, resource_pool_name=None):
             return self.system.get_obj(vim.ResourcePool, self.system.default_resource_pool)
         return self.system.get_obj_list(vim.ResourcePool)[0]
 
-    def _clone(self, destination, resourcepool=None, datastore=None, power_on=True,
-               sparse=False, template=False, provision_timeout=1800, progress_callback=None,
+    def _get_cluster_compute_resource(self, cluster_compute_resource_name=None):
+        """ Returns a Compute Cluster Resource managed object. If a name is specified,
+        a vim.ClusterComputeResource object is returned for the specific resource. If no name if
+        specified, the method checks if there are is a default resource specified and returns the
+        object of the resource. Finally,  if there is no name or defaults specified, it queries
+        the Vmware provider and picks the first object returned in the list.
+
+        Args:
+            cluster_compute_resource_name (string): The name of the Cluster Compute Resource.
+            If None, first one will be picked.
+        Returns:
+             pyVmomi.vim.ClusterComputeResource: The managed object of the cluster compute resource.
+        """
+        if cluster_compute_resource_name is not None:
+            return self.system.get_obj(vim.ClusterComputeResource, cluster_compute_resource_name)
+        elif self.system.default_cluster_compute_resource is not None:
+            return self.system.get_obj(vim.ClusterComputeResource,
+                                       self.system.default_cluster_compute_resource)
+        return self.system.get_obj_list(vim.ClusterComputeResource)[0]
+
+    def _set_vm_relocate_spec(self, resourcepool, host, sparse, progress_callback,
+                              deploy_on_ds_cluster):
+        """Set properties for Virtual Machine Relocate Operation specification """
```

Comment:
Thoughts on having a docblock here? Some of these variables are not totally clear to me, like `sparse`.

Reply:
I think it would be more appropriate to define this within the `_clone` method since these variables are just passed down. Sparse is a type of disk format when copying a VM